### DISCUSSION TITLE
The future of Iran: designing a horizontal system for political decision making

### DISCUSSION THREAD

**Discussion by Zohreh-sudo** [WeAreMahsaAmini/Core#457] [Ideas]
The idea is related to post-revolution, but it’s time to start and test its fundamental now. It could indeed accelerate revolution as well. It’s a social network for Iranians everywhere including Iranian experts in all fields to network and team up based on one „Baraye“ and brainstorm together for problem-solving. It follows the idea for „the responsible citizen“ collecting  their creativity and expertise together to find blandly new ideas for the future of Iran. Hopefully the system could set-up a successful horizontal democratic system in the future.
I‘d love to work on this project in your repository. I‘m just totally new at GitHub and would love to build a team with different expertise. Would you support me?

**Comment by tcf-admin** [comment]
Hi @Zohreh-sudo, 

we have researched different electoral system and participation methods that can be implemented in e-democracy platforms. 
Currently we are implementing majority, plurality and condorcet methods into the participation feature of forDem, an e-democracy platform: https://github.com/tcfev/forDem
Please check it out and let us know if you are interested in collaboration.
